packages/suite/src/services/coinjoin/config.ts

```diff
@@ -1,16 +1,54 @@
-const REGTEST_URL = 'https://dev-coinjoin.trezor.io/'; // 'https://coinjoin-dev.sldev.cz/'; // 'http://localhost:8081/';
+import type { CoinjoinBackendSettings, CoinjoinClientSettings } from '@trezor/coinjoin';
+import type { PartialRecord } from '@trezor/type-utils';
+import type { CoinjoinServerEnvironment } from '@suite-common/wallet-types';
+import type { NetworkSymbol } from '@wallet-types';
 
-export const COINJOIN_NETWORKS: Record<string, any> = {
+type CoinjoinNetworksConfig = CoinjoinBackendSettings &
+    CoinjoinClientSettings & {
+        percentageFee: string;
+    };
+
+type ServerEnvironment = PartialRecord<CoinjoinServerEnvironment, CoinjoinNetworksConfig>;
+
+export const COINJOIN_NETWORKS: PartialRecord<NetworkSymbol, ServerEnvironment> = {
     regtest: {
-        network: 'regtest',
-        coordinatorName: 'CoinJoinCoordinatorIdentifier',
-        coordinatorUrl: `${REGTEST_URL}WabiSabi/`,
-        // backend settings
-        blockbookUrls: [`${REGTEST_URL}blockbook/api/v2`],
-        baseBlockHeight: 0,
-        baseBlockHash: '0f9188f13cb7b2c71f2a335e3a4fc328bf5beb436012afca590b1a11466e2206',
-        // client settings
-        middlewareUrl: `${REGTEST_URL}Cryptography/`,
-        percentageFee: '0.03',
+        public: {
+            network: 'regtest',
+            coordinatorName: 'CoinJoinCoordinatorIdentifier',
+            coordinatorUrl: 'https://dev-coinjoin.trezor.io/WabiSabi/',
+            // backend settings
+            blockbookUrls: ['https://dev-coinjoin.trezor.io/blockbook/api/v2'],
+            baseBlockHeight: 0,
+            baseBlockHash: '0f9188f13cb7b2c71f2a335e3a4fc328bf5beb436012afca590b1a11466e2206',
+            // client settings
+            middlewareUrl: 'https://dev-coinjoin.trezor.io/Cryptography/',
+            // suite settings
+            percentageFee: '0.03',
+        },
+        localhost: {
+            network: 'regtest',
+            coordinatorName: 'CoinJoinCoordinatorIdentifier',
+            coordinatorUrl: 'http://localhost:8081/WabiSabi/',
+            // backend settings
+            blockbookUrls: ['http://localhost:8081/blockbook/api/v2'],
+            baseBlockHeight: 0,
+            baseBlockHash: '0f9188f13cb7b2c71f2a335e3a4fc328bf5beb436012afca590b1a11466e2206',
+            // client settings
+            middlewareUrl: 'http://localhost:8081/Cryptography/',
+            // suite settings
+            percentageFee: '0.03',
+        },
     },
 };
+
+export const getCoinjoinConfig = (
+    network: NetworkSymbol,
+    environment?: CoinjoinServerEnvironment,
+) => {
+    const config = COINJOIN_NETWORKS[network];
+    const settings = config ? config[environment ?? 'public'] : undefined;
```

Comment:
nitpic/note/no need to change... 

You could also type it using optional chaining...
```suggestion
    const settings = config?.[environment ?? 'public'];
```